### ISSUE TITLE
Occasional Black screen using GameCube biis

### ISSUE THREAD

**Issue by WizLizard** [FIX94/Nintendont#346]
Sometimes booting a game using the gamecube bios i will get a black screen with audio. The game is still running but everything is black. I can hit tee home button to return to usb loader gx or nintedont but everything remains black. I haven't been able to reproduce the issue when not using a bios.

**Comment by carnage702** [comment]
yes this is a known issue but its so random it very very hard to find the issue, if you dont use the bios it will never happen, also afaik it only happens on vwii not on the wii for some reason.

**Comment by WizLizard** [comment]
Yes you are right i have only experienced this on the vwii. It would be
nice to get it fixed for vwii. I like having the option to see the bios,
it's so nostalgic. Makes it seem like a genuine gamecube as well. But this
random black screen is annoying. I would greatly appreciate it if someone
could find the problem and resolve it. I would try myself but I'm not smart
enough lol.

**Comment by DongVGH** [comment]
This is not vWii exclusive, it also occurs on Wii.

**Comment by vbarros22** [comment]
I have the same problem in WII, but it has been happening because of a generic Wii2HDMI that I buy, the problem comes even in games that have native progressive scan mode support, but it resolves when active. I did not expect it to happen on Wii U, since HDMI is official. Well, I solve this by activating progressive mode for ALL games, I hope it works for you as well, since this is a problem with video output and TV support, not Nintendont. I had a component and had no problem, since with this cable, in addition to random black screens, I can not play the NeoGeo games in the Virtual Console, and Megaman X Collection, gives a black screen that has no way to reverse (before the problem It was my old TV, that even with the component cable, this black screen happened because it did not support 240p, but this new one has, so the only possible problem is the generic cable).
Sorry for my fluent English. LOL

**Comment by carnage702** [comment]
this has nothing to do with porgressive or cables ... its a known issue that sometimes vwii will not show video when using the GC bios, wich is just an addon and totaly optional, if you remove the bios it will never blackscreen on boot randomly.
